scripts/NOTES.md

```diff
@@ -9,3 +9,13 @@
 7. Run `heuristic_metadata.py` to generate `metadata-joined.csv`.
 8. Run `run_nx.py` to generate `metadata-final.csv`. This is the file that will be used to populate the PostgreSQL database.
 9. Run `combine_metadata.py` to add clusters to `metadata-pruned.csv`.
+
+
+# Order of Operations for downloading data
```

Comment:
does this bring data into the local database? or is this something else

Reply:
This is for downloading raw ethereum transactions into the compute cluster, where we process it before putting into db.